packages/wonder-blocks-modal/components/one-column-modal.md

```diff
@@ -0,0 +1,119 @@
+**One-Column, with footer:**
+
+A one-column modal dialog with a footer on the contents. The contents expand to fill the container and it continues to grow in size. There is no minimum height to the modal.
+
+```js
+const {StyleSheet, css} = require("aphrodite");
+const {View} = require("wonder-blocks-core");
+const {Title, Body} = require("wonder-blocks-typography");
+const OneColumnModal = require("./one-column-modal.js").default;
+
+const styles = StyleSheet.create({
+    example: {
```

Comment:
I updated some of these styles in #64, so we'll probably want to keep that in mind when we merge these changes together. (Maybe it's approaching time to actually pull them into a shared example component!)

Reply:
Good idea! I'll wait on you and I can go back through and update these once landed.